documentation/tutorials/sensitivity/PEcAn_sensitivity_tutorial_v1.0.Rmd

```diff
@@ -101,46 +107,40 @@ You should see a list of long numbers. Each of these is an ID. In the Files pane
 
 Let’s try to read the output from a single run id, as you did in the earlier tutorial. **read.output** is a PEcAn function to read in variables from any PEcAn compliant output files, it requires a runid, the directory, start year, end year, and an optional list of variables. Make sure you have RunDir variable set to the run number and the year to the year you ran.
 ```{r,eval=FALSE}
```

Comment:
Line 108: "from any PEcAn compliant output files, it requires a runid," -> "from any PEcAn compliant output files, **and** it requires a runid,"